### ISSUE TITLE
Broken sdist on PyPI

### ISSUE THREAD

**Issue by Zuzu-Typ** [ranjian0/assimp-py#1]
Hello there (:

When trying to install assimp_py from the PyPI, I've encountered a problem.
Since there are no wheels for Python 3.9, I tried downloading the source distribution (assimp_py-1.0.2.tar.gz) and install it via 
`python setup.py install`.
However, the command crashes with the following output:
```cmd
C:/some/folder/assimp_py-1.0.2> python setup.py install
running install
running bdist_egg
running egg_info
writing assimp_py.egg-info\PKG-INFO
writing dependency_links to assimp_py.egg-info\dependency_links.txt
writing top-level names to assimp_py.egg-info\top_level.txt
reading manifest file 'assimp_py.egg-info\SOURCES.txt'
Traceback (most recent call last):
  File "C:/some/folder/assimp_py-1.0.2/setup.py", line 84, in <module>
    main()
  File "C:/some/folder/assimp_py-1.0.2/setup.py", line 63, in main
    setup(name="assimp_py",
  File "C:\Python39\lib\distutils\core.py", line 148, in setup
    dist.run_commands()
  File "C:\Python39\lib\distutils\dist.py", line 966, in run_commands
    self.run_command(cmd)
  File "C:\Python39\lib\distutils\dist.py", line 985, in run_command
    cmd_obj.run()
  File "C:\Python39\lib\site-packages\setuptools\command\install.py", line 67, in run
    self.do_egg_install()
  File "C:\Python39\lib\site-packages\setuptools\command\install.py", line 109, in do_egg_install
    self.run_command('bdist_egg')
  File "C:\Python39\lib\distutils\cmd.py", line 313, in run_command
    self.distribution.run_command(command)
  File "C:\Python39\lib\distutils\dist.py", line 985, in run_command
    cmd_obj.run()
  File "C:\Python39\lib\site-packages\setuptools\command\bdist_egg.py", line 158, in run
    self.run_command("egg_info")
  File "C:\Python39\lib\distutils\cmd.py", line 313, in run_command
    self.distribution.run_command(command)
  File "C:\Python39\lib\distutils\dist.py", line 985, in run_command
    cmd_obj.run()
  File "C:\Python39\lib\site-packages\setuptools\command\egg_info.py", line 293, in run
    self.find_sources()
  File "C:\Python39\lib\site-packages\setuptools\command\egg_info.py", line 300, in find_sources
    mm.run()
  File "C:\Python39\lib\site-packages\setuptools\command\egg_info.py", line 531, in run
    self.add_defaults()
  File "C:\Python39\lib\site-packages\setuptools\command\egg_info.py", line 575, in add_defaults
    self.read_manifest()
  File "C:\Python39\lib\site-packages\setuptools\command\sdist.py", line 190, in read_manifest
    self.filelist.append(line)
  File "C:\Python39\lib\site-packages\setuptools\command\egg_info.py", line 473, in append
    path = convert_path(item)
  File "C:\Python39\lib\distutils\util.py", line 123, in convert_path
    raise ValueError("path '%s' cannot be absolute" % pathname)
ValueError: path '/Users/runner/work/assimp_py/assimp_py/extern/assimp/code/CApi/AssimpCExport.cpp' cannot be absolute
```

The cause of this error is the `SOURCES.txt` file in the `.egg-info` folder inside the archive.
`SOURCES.txt` contains entries like:
```
/Users/runner/work/assimp_py/assimp_py/extern/assimp/code/CApi/AssimpCExport.cpp
/Users/runner/work/assimp_py/assimp_py/extern/assimp/code/CApi/CInterfaceIOWrapper.cpp
...
```
These are the absolute paths used when creating the source distribution. It should use relative paths.

The issue has already been fixed on the master branch, but it has not made it's way into a release yet. The latest release is [33 commits behind master](https://github.com/ranjian0/assimp_py/compare/v1.0.2...master).
--Edit--
Well, creating an sdist is fixed on the master branch, but I was not able to build it. Could be a Windows / Py3.9 related error.
It is possible to build the version from the PyPI, by simply removing the `.egg-info` folder.
-/Edit--

It would be very nice if you could publish a new release with these changes inside.
While you're at it, you might want to update the supported Python versions in [setup.py](https://github.com/ranjian0/assimp_py/blob/master/setup.py#L93) and the [GitHub Actions workflows](https://github.com/ranjian0/assimp_py/tree/master/.github/workflows).
I have not used Actions myself yet, otherwise I would have provided you with a PR.

Oh, and one more side note, you don't seem to have a LICENSE file in your project, despite [stating](https://github.com/ranjian0/assimp_py/blob/master/setup.py#L89) your project uses an MIT license. It's easy to add the license to your project, by hitting `Add file`, putting "LICENSE" or "LICENSE.md" as the file name and selecting `Choose a license template`. There you'll find the MIT license, ready to guard your copyright (;

Cheers
--Zuzu_Typ--

**Comment by ranjian0** [comment]
Hi, sorry for the late reply, was quite busy.

1. First of all, I had completely given up on this esp getting windows version to build. I don't have a windows machine and I was relying on github actions to debug windows builds which was very slow and frustrating.

2. I will do my best to get it up and running for all but no guarantees.

**Comment by Zuzu-Typ** [comment]
Hi there (: 

don't worry about the late reply.

I think I fixed it by the way. [These changes](https://github.com/Zuzu-Typ/assimp_py/tree/test-win-build) should do the trick (at least I got it to build on my machine now). 

I could create a PR if you'd like.

Cheers,
--Zuzu_Typ--

**Comment by ranjian0** [comment]
Since I am already working on it let me make the changes.

Thanks.

**Comment by ranjian0** [comment]
Closed by #2 

Thanks.

**Comment by Zuzu-Typ** [comment]
Awesome!

Thank you for your work on the fix ^^
And for creating the library in general for that matter :]
